.storybook/preview.tsx

```
@@ -15,9 +15,22 @@ const queryClient = new QueryClient({
 
 const preview: Preview = {
   decorators: [
-    (Story) => (
+    (Story, context) => (
       <QueryClientProvider client={queryClient}>
-        <Story />
+        {context.viewMode === 'story' ? (
+          <div
+            style={{
+              width: '100%',
```

ESLint

[error] Replace `'100%'` with `"100%"`

(prettier/prettier)